src/uas/UAS.cc

```diff
@@ -2827,21 +2825,6 @@ void UAS::enableHilXPlane(bool enable)
         }
         qDebug() << "CREATED NEW XPLANE LINK";
         simulation = new QGCXPlaneLink(this);
-
-        float noise_scaler = 0.1f;

```

Comment:
@scott-eddy Here's a log showing an attempted stall sequence with the Rascal110 in FlightGear. Although I did see a stall break this time, it doesn't behave at all like my real Rascal...
https://copy.com/DfK4d90BNu0p3UTk

I'm looking forward to seeing your simulation log.